SitumWayfinding/Classes/Positioning/Navigation/InfoBarNavigationViewController.swift

```diff
@@ -121,3 +100,68 @@ extension SITNavigationProgress {
         }
     }
 }
+
+extension InfoBarNavigationViewController {
+    func prepareElements() {
+        if #available(iOS 13.0, *) {
```

Comment:
Poderias simplificala como che comentei máis arriba? Comprobando so se hai que aplicar modo escuro ou non?
Aquí se non o hai que aplicar podes comprobar se é ios13 ou posterior e usar as imaxes por defecto ou facelo como se facia antes. Pero se creas dous métodos vaiche quedar o código moito máis limpo e claro